src/utils/transformer-utils.ts

```diff
@@ -98,9 +79,9 @@ export const traversePortableText = <T extends ArbitraryTypedObject = PortableTe
 };
 
 export const createSpan = (
-  guid: string,
+  guid: ShortGuid,
   marks?: string[],
-  text?: string
+  text?: string,
 ): PortableTextSpan => {
   return {
     _type: "span",
```

Comment:
refactor nitpick: these functions don't need to return in block :) can save a few lines but its not critical :D